walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/datasource/network/apis/PresentationApis.kt

```diff
@@ -34,6 +36,7 @@ internal interface PresentationApis {
         @Url overrideUrl: String,
         @Field("id_token") token: String,
         @Field("vp_token") vpToken: List<String>,
-        @Field("state") state: String?
+        @Field("state") state: String?,
+        @HeaderMap additionalHeaders: Map<String, String>,
```

Comment:
does @HeaderMap just append the new headers to existing headers or create a new map?

Reply:
Yes, according to the docs, 
> Adds headers specified in the Map or okhttp3.Headers.
> Values in the map are converted to strings using Retrofit.stringConverter(Type, Annotation[]) (or Object.toString(), if no matching string converter is installed).